### PR TITLE
feat(CLI): add accessToken flag to CLI

### PR DESCRIPTION
**Pull request by hisabimbola**
#### Summary

Add the accessToken flag to the CLI
#### TODO

- Tests
    - [x] Unit
    - [x] Integration
    - [x] Acceptance
- [ ] Documentation

### CODE REVIEW

File: features/step_definitions/cli_steps.js [hunk 1]
```diff
@@ -12,7 +17,7 @@ export default function cliSteps () {
   mkdirp(tmpDir)
 
   function uniqueId (prefix) {
-    const id = `${++idCounter}`
+    const id = `${++idCounter}` // eslint-disable-line no-plusplus
```
Comment (Siilwyn) [inline features/step_definitions/cli_steps.js]: I actually like this rule. Should be `${idCounter += 1}`

File: features/step_definitions/cli_steps.js [hunk 2]
```diff
@@ -53,12 +58,12 @@ export default function cliSteps () {
 
       // replace placeholder for generate unique ids
       const replacedFileContent = fileContent
-        .replace(/\<id\-(\w)\>/gi, (match, g1) => uniqueId(g1))
+        .replace(/<id-(\w)>/gi, (match, g1) => uniqueId(g1))
 
-      mkdirp(dirName, err1 => {
+      mkdirp(dirName, (err1) => {
```
Comment (Siilwyn) [inline features/step_definitions/cli_steps.js]: err1 > directoryWriteError
err2 > fileWriteError
Maybe? Doing this for the linter I assume?

Comment (hisabimbola) [inline features/step_definitions/cli_steps.js]: Yep 😉

File: features/step_definitions/cli_steps.js (hunk 1 above)
Comment (hisabimbola) [inline features/step_definitions/cli_steps.js]: It's actually different `++idCounter` !== `idCounter += 1`, but will trying to see if I can get something close

File: features/step_definitions/cli_steps.js [hunk 3]
```diff
@@ -79,13 +85,43 @@ export default function cliSteps () {
     })
   })
 
+  this.When(
+    /^I run with accessToken `sphere(| .+)`$/,
```
Comment (emmenko) [inline features/step_definitions/cli_steps.js]: Why exactly do you need this for? To get a "real" access token?

Comment (hisabimbola) [inline features/step_definitions/cli_steps.js]: Yeah, to get a real access token from the API, for the cucumber test to work.

**Pull request merged**